src/components/AppNavigation/Settings/SettingsAddressbook.vue

```diff
@@ -107,7 +107,15 @@
 				</template>
 
 				<!-- delete addressbook -->
-				<ActionButton v-if="hasMultipleAddressbooks"
+				<ActionButton v-if="hasMultipleAddressbooks && !addressbook.url.includes(addressbook.owner)"
```

Comment:
What is the idea behind `!addressbook.url.includes(addressbook.owner)`?

Reply:
nvm, I got it wrong, I will try and find the principal uri somehow.